tests/test_fields.py

```diff
@@ -249,3 +249,30 @@ def test_empty_field():
     assert not empty.check_cell('')
     with pytest.raises(AssertionError):
         assert empty.check_cell('1')
+
+
+def test_custom_message():
+    f = field.Text(1, custom_error_msg='CustomErrorMessage')
+    try:
```

Comment:
```python
with pytest.raises(AssertionError):
    f.check_cell('')
```

Reply:
Try `_has_error` again, it shouldn't raise the AssertionError so pytest won't bomb out